src/System.Private.DeveloperExperience.Console/src/Internal/DeveloperExperience/ConsolePal.Unix.cs

```diff
@@ -9,7 +10,7 @@ internal static class ConsolePal
     {
         internal unsafe static void WriteError(string errorMessage)
         {
-            byte[] errorMessageAsBytes = Interop.StringHelper.GetBytesFromUTF8string(errorMessage);
+            byte[] errorMessageAsBytes = Encoding.UTF8.GetBytes(errorMessage);
```

Comment:
This is actually fixing a bug. GetBytesFromUTF8string returned zero-terminate string, but we do not actually want the zero terminator here.